storage/client/blob/CHANGELOG.md

```diff
@@ -0,0 +1,115 @@
+# Change Log azure-storage-blob
+
+## Version 12.0.0-preview.1:
+
+Version 12.0.0-preview.1 is the first preview of our efforts to create a user-friendly and Java client library for Azure Storage Blobs. For more information about this, and preview releases of other Azure SDK libraries, please visit
+https://aka.ms/azure-sdk-preview1-java.
+
+**Breaking changes: New API design**
+- Operations are now scoped to a particular client:
+    - `StorageClient`: This client handles account-level operations. This includes managing service properties and listing the containers within an account.
+    - `ContainerClient`: The client handles operations for a particular container. This includes creating or deleting that container, as well as listing the blobs within that container and managing properties and metadata.
```

Comment:
I think we should flip the last parts about metadata and properties and listing blobs in the container, right now it reads a bit like this client manages properties and metadata of the blobs it can list

Reply:
Removed the manageProperty and metadata.